test/swar/BasicOperations.cpp

```diff
@@ -68,7 +68,7 @@ using S4U8 = SWAR<4, u8>;
 constexpr auto A = S4U8{Literals<4, u8>, {4, 4}};
 constexpr auto B = std::array<u8, 2>{4, 4};
 
-static_assert(compareContents<S4U8::Lanes, std::array<u8, 2>>(A.to_array(), B));
+static_assert(compareContents<S4U8::Lanes>(A.to_array(), B));
```

Comment:
I don't think giving the names `A` and `B` help clarity in any way.
It seems to me the defining expressions for `A` and `B` are clearer than splitting them into a `A` and `B`.